discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -228,10 +226,9 @@ def validate_track_tx(params: ManageEntityParameters):
                 f"Cannot create track {track_id} below the offset"
             )
     if params.action == Action.CREATE or params.action == Action.UPDATE:
-        track_metadata = params.metadata.get(params.metadata_cid)
-        if track_metadata is not None:
-            track_bio = track_metadata.get("description")
-            track_genre = track_metadata.get("genre")
+        if params.metadata:
```

Comment:
`track_metadata = params.metadata.get(params.metadata_cid)` should just be `params.metadata`